requirements.txt

```diff
@@ -250,3 +250,4 @@ WTForms==2.2.1
 WTForms-Alchemy==0.16.9
 WTForms-Components==0.10.3
 zenodo-accessrequests==1.0.0a5
+joblib==0.14.1
```

Comment:
I think we also need the `scikit-learn` dependencies for the model to work, otherwise, it would complain with import errors.

Reply:
I thought so too but that is not the case, it seems like scikit is building a standalone python code.